libsplinter/src/rest_api/auth/roles/store/mod.rs

```diff
@@ -14,13 +14,19 @@
 
 //! This module defines the store trait for roles and their assignments to identities.
 
-#[cfg(feature = "sqlite")]
+#[cfg(any(
+    feature = "role-based-authorization-store-postgres",
+    feature = "sqlite"
+))]
```

Comment:
These should just be #[cfg(feature = "diesel")].

Reply:
Fixed